scripts/scil_remove_invalid_streamlines.py

```diff
@@ -63,8 +65,7 @@ def main():
     if args.threshold < 0:
         parser.error("Threshold must be positive.")
 
-    sft = load_tractogram_with_reference(parser, args, args.in_tractogram,
```

Comment:
Considering the nature of the script, I would leave as it was.
100% of the time a user will want the check OFF if they want to remove invalid.
So I would revert this one.